plasmapy/__init__.py

```diff
@@ -16,12 +16,8 @@
            "core Python package for plasma physics.")
 
 
-class UnsupportedPythonError(Exception):
-    pass
-
-
 if sys.version_info < tuple((int(val) for val in "3.6".split('.'))):
-    raise UnsupportedPythonError("plasmapy does not support Python < {}".format(3.6))
+    raise Exception("plasmapy does not support Python < {}".format(3.6))
```

Comment:
What would you think of using an `ImportError` here, or perhaps a `RuntimeError`?

Reply:
I keep forgetting all the exceptions available. `Runtime` sounds good!